lib/connextclient/ConnextClient.ts

```diff
@@ -692,6 +694,44 @@ class ConnextClient extends SwapClient {
     });
   }
 
+  public async withdraw({
```

Comment:
Mainly stylistic in this case but we're using arrow functions inside classes so this ought to be `public async withdraw = ...`